src/pages/predictions.vue

```diff
@@ -0,0 +1,416 @@
+<template>
+  <Layout>
+    <!-- Navigation-->
+    <nav class="navbar navbar-expand-lg navbar-dark fixed-top py-1" id="mainNav">
+      <a class="navbar-brand font-weight-light js-scroll-trigger" href="/">
+        <h1 class="main-h1">Prediction Graphs</h1>
+      </a>
+
+      <div class="list-js-search">
+        <label class="search-label-hidden" for="search">Search by country</label>
+        <input class="search" type="text" id="search" placeholder="Search by country" />
+        <i class="fas fa-search-location"></i>
+      </div>
+    </nav>
```

Comment:
Create this as a component (title and search) and re-use in both index and predictions.